.github/workflows/release.yml

```diff
@@ -142,14 +142,20 @@ jobs:
         uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERIO_USERNAME }}
-          password: ${{ secrets.DOCKERIO_TOKEN }}    
+          password: ${{ secrets.DOCKERIO_TOKEN }}           
+      - name: Remove v- prefix
+        id: rmprefix
+        env:
+          VERSION: ${{ github.ref_name }}        
+        run: |
+          echo "${VERSION:1}"
       - name: Build and push
         uses: docker/build-push-action@v3
         with:
           push: true
           file: sql/Dockerfile
           platforms: linux/amd64,linux/arm64
           context: sql/
-          tags: tensorchord/envd-server-migration:${{ github.ref_name }}
+          tags: tensorchord/envd-server-migration:${{ steps.rmprefix.outputs.value }}
```

Comment:
Please also update the manifest/values.yaml

Reply:
It used the same tag as the server image tag, which is right after this PR merged

Reply:
https://github.com/tensorchord/envd-server/blob/0ad4c85ebb858397a19c2139cf1107e0567f6b02/manifests/templates/deployment.yaml#L6

Reply:
fixed